modules/base/src/components/Base.vue

```diff
@@ -164,6 +169,58 @@ export default {
       return this.uiModules.filter(value => {
         return value.bottom;
       });
+    },
+    activeReleaseLevel() {
+      // get db record
+      var record = this.configDB.records["config:core/releaseLevel"];
+      if (!record) {
+        return 0; // ReleaseLevelStable
+      }
+
+      // get active value
+      var value = record.DefaultValue;
+      if (record.hasOwnProperty("Value") && record.Value !== null) {
+        value = record.Value;
+      }
+
+      switch (value) {
+        case "stable": // ReleaseLevelNameStable
+          return 0; // ReleaseLevelStable
+        case "beta": // ReleaseLevelNameBeta
+          return 1; // ReleaseLevelBeta
+        case "experimental": // ReleaseLevelNameExperimental
+          return 2; // ReleaseLevelExperimental
+        default:
+          return 0; // ReleaseLevelStable
+      }
```

Comment:
I guess constants would be useful for those numbers

Reply:
Have not yet figured out how to do constants in a nice way with Vue.js.